src/benchmarklib/benchmark_runner.cpp

```diff
@@ -259,27 +264,37 @@ void BenchmarkRunner::_benchmark_ordered() {
     }
     _state.set_done();
 
+    // Wait for the rest of the tasks that didn't make it in time - they will not count toward the results
+    std::cout << "  -> Waiting for clients that are still running" << std::endl;
```

Comment:
Could you check that `_currently_running_clients > 0`? Otherwise, we end up printing this in single-threaded mode, too.